terraform/modules/paas/data.tf

```diff
@@ -3,7 +3,7 @@ data cloudfoundry_org org {
 }
 
 data cloudfoundry_space space {
-  name = "bat-${var.app_environment}"
+  name = var.app_environment == "sandbox" ? "bat-prod" : "bat-${var.app_environment}"
```

Comment:
Can we add some kind of lookup table instead in variables? It would match the environment with the target space

Reply:
have added space as a variable with values defined in the {env}.tfvars file.